### DISCUSSION TITLE
Ember Waste Expansion

### DISCUSSION THREAD

**Discussion by OcelotWalrus** [OcelotWalrus/Cromha-Expansion-plugin#118] [General]
The Ember Waste is an interesting region of the Milky Way. There is already some expansion to that region but none of them are very important. We will talk about anything that is in rapport with the Ember Waste here. You can propose ideas so they are noted here:

**TO-DO**:
- [ ] Add more Remnant systems
- [ ] Expand the Graveyard
- [x] Give the Ka'het faction new ships and maybe some new systems (ships: `images/ship/new/veliante'ixis/veliante'ixis`
- [ ] Expand the lore of the Builders
- [ ] Expand the Remnant storyline that is already in the game
- [ ] At to the storyline a relic: The Korath Ancient Raider (sprite: `images/ship/new/ancient raider`
- [ ] Add a variant of the Albatross using `images/ship/new/prefect's albatross` sprite
- [x] Add a new person that will be a Remnant that patrol in the Graveyard (sprite: `images/ship/new/ember nightmare/ember nightmare`
- [x] Add a new Remnant ship: the Nightingale. It will be a transport class ship that will be used by both Remnant and Far Remnant

**Comment by lumbar527** [comment]
> - [ ]  Add a new Remnant ship: the Nightingale. It will be a transport class ship that will be used by both Remnant and Far Remnant

Is there a sprite for this ship? If not do you think I could try getting it?

**Reply by OcelotWalrus** [reply]
There isn't any. You can try to make/find one.

**Reply by lumbar527** [reply]
Thanks! I'll look into that!

**Reply by lumbar527** [reply]
I've created some potential stuff you can use, see OcelotWalrus/Cromha-Expansion-sources#1

**Reply by OcelotWalrus** [reply]
I've checked that out. It looks really nice. You can make the ship if you want.

**Reply by lumbar527** [reply]
Thanks; I'll look at doing that.

**Reply by lumbar527** [reply]
I forgot to do fleets and shipyards but you can test out the ship with this commit: 108b14d1091ee6c8bb084ec5587dfab992d3e24e

**Reply by OcelotWalrus** [reply]
Thanks for working on it!

**Reply by lumbar527** [reply]
You're welcome!
